### ISSUE TITLE
Drop Down remains the same even when selected other label.

### ISSUE THREAD

**Issue by mohitms007** [Girl-Code-It/Opportunity-Calendar-Frontend#45]
The Opportunities drop down remains the same even if we select for example scholarships it doesn't change according to the item we selected.

**Comment by mohitms007** [comment]
Can I work on this?

**Comment by abdus** [comment]
@mohitms007 I have assigned you the other issue. It's against the rules to assign multiple issues at the same time.

**Comment by mohitms007** [comment]
Sorry didn't knew this. Thanks!

**Comment by vismitap** [comment]
May I work on this issue as a part of GSSOC'21?

**Comment by healer9** [comment]
Hello, @abdus I also want to work on this issue, if it didn't get resolved.

**Comment by vismitap** [comment]
Hey! I did a git pull and got the latest code and the on `npm start` I got this error. 
![image](https://user-images.githubusercontent.com/47056243/111041172-0a072f00-845d-11eb-83e3-c4a6b48bfcee.png)
I inspected the folder directory and noticed that there are 2 signUpLogin folders having same code. 
![image](https://user-images.githubusercontent.com/47056243/111041247-5a7e8c80-845d-11eb-8a0c-5b5bd77f28ba.png)

Which folder should I delete? What should I do next?
